src/main/resources/templates/nonceStateCheck.html

```diff
@@ -67,8 +72,7 @@ <h5 class="alert alert-warning">In a real tool that is not for demonstration pur
 
         <div class="alert alert-info">
             <p>
-                The LTI tool is using the <code>target</code> to retrieve the <code>nonce</code> and <code>state</code> from
-                the storage.
+                The LTI tool is retrieving the <code>nonce</code> and <code>state</code> from the <code>target</code> frame storage.               the storage.
```

Comment:
Is there an extra "the storage" at the end of this line that needs to be removed?